Kitodo/src/main/java/de/sub/goobi/config/Parameters.java

```diff
@@ -0,0 +1,695 @@
+     */
+    public static final String DIRECTORY_SUFFIX = "DIRECTORY_SUFFIX";
+
+    /**
+     * Boolean, defaults to {@code false}.
+     */
+    public static final String IMPORT_USE_OLD_CONFIGURATION = "importUseOldConfiguration";
+
+    /**
+     * Creation and export of process sub-directories, e.g.
+     * {@code images/(processtitle)_tif&ocr/(processtitle)_pdf}.
+     * {@code (processtitle)} is a placeholder for the process title If you
+     * comment in the parameter processDirs without a value, the result is that
+     * the whole process directory will be exported and no directory well be
+     * created. If you leave the parameter commented out, the whole
+     * functionality is disabled. Using the {@code processDirs} parameter is
+     * always an addition to the existing folder creating and exporting
+     * functions of Kitodo.Production.
+     */
+    public static final String PROCESS_DIRS = "processDirs";
+
+    /**
+     * Set if master images folder {@code orig_} should be used at all. Boolean,
+     * defaults to {@code true}.
+     */
+    public static final String USE_ORIG_FOLDER = "useOrigFolder";
+
+    // -----------------------------------
+    // Directory and symbolic link management
+    // -----------------------------------
+
+    /**
+     * Script to create the user's home directory when adding a new user.
+     */
+    public static final String SCRIPT_CREATE_DIR_USER_HOME = "script_createDirUserHome";
+
+    /**
+     * Script to create the directory for a new process.
+     */
+    public static final String SCRIPT_CREATE_DIR_META = "script_createDirMeta";
+
+    /**
+     * Script to create a symbolic link in the user home direcory and set
+     * permissions for the user.
+     */
+    public static final String SCRIPT_CREATE_SYMLINK = "script_createSymLink";
+
+    /**
+     * Script to remove the symbolic link from the user home directory.
+     */
+    public static final String SCRIPT_DELETE_SYMLINK = "script_deleteSymLink";
+
+    // -----------------------------------
+    // Runnotes
+    // -----------------------------------
+
+    /**
+     * Filename of the XSLT file for transforming old metadata files which need
+     * to be in the xslt folder above.
+     */
+    public static final String XSLT_FILENAME_METADATA_TRANSFORMATION = "xsltFilenameMetadataTransformation";
+
+    // -----------------------------------
+    // Images
+    // -----------------------------------
+
+    /**
+     * Prefix for image names as regex. Default is 8 digits \\d{8} and gets
+     * validated.
+     */
+    public static final String IMAGE_PREFIX = "ImagePrefix";
+
+    /**
+     * Sorting of images.
+     */
+    public static final String IMAGE_SORTING = "ImageSorting";
+
+    /**
+     * Sorting of images: 1 is lesser then 002, compares the number of image
+     * names, characters other than digits are not supported
+     */
+    public static final String IMAGE_SORTING_VALUE_NUMBER = "number";
```

Comment:
Here you are mixing config keys and values. Shouldn't they be separated?

Reply:
These are value constants allowed to be used with the corresponding key. I am using the pattern `<key-name>_VALUE_<constant-name>` for this. This clarifies how things belong together.